src/jdk.compiler/share/classes/com/sun/tools/javac/resources/javac.properties

```diff
@@ -168,16 +168,16 @@ javac.opt.Xbootclasspath.p=\
 javac.opt.Xbootclasspath.a=\
     Append to the bootstrap class path
 javac.opt.Xlint=\
-    Enable recommended warnings
+    Enable recommended lint warning categories
```

Comment:
I think the use of "mandatory warning" above is good. However, I find the use of `lint warning` a bit too "implementation-related". I believe I liked the old text better, even if it was more vague.

Reply:
Related, as part of this PR it might be needed to do a pass on the man pages, to make sure they are in sync -- I see for instance this:
```
       -nowarn
              Disables warning messages. This option operates the same as the -Xlint:none option.
```
Which would no longer be valid after this change. E.g.

```
javac -Xlint:none -Xlint:serial
```
is not the same as:
```
javac -nowarn -Xlint:serial
```

Reply:
I also wonder if the combo `nowarn` + (non-empty lint set) should result in an option warning (as the lint directive will be skipped) ?

Reply:
I agree with your verbiage comments and also thanks for the reminder about the man page.

Both issues should be addressed in ffede9b0508.

>  I also wonder if the combo nowarn + (non-empty lint set) should result in an option warning (as the lint directive will be skipped) ?

I don't think so. For example, it's probably common for people to throw a temporary `-nowarn` into their build process while they work on something. You don't want to have to then _also_ temporarily find and remove the `-Xlint` flags that are normally there (and then remember later to also un-do that).

Put more simply, there's something fishy about having the simple addition of a `-nowarn` flag cause a new warning to appear :)

Reply:
> Put more simply, there's something fishy about having the simple addition of a `-nowarn` flag cause a new warning to appear :)

That is true :-)